ci-operator/jobs/openshift/hive/openshift-hive-master-presubmits.yaml

```diff
@@ -255,6 +255,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: build02
+    context: ci/prow/e2e-pool
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-hive-master-e2e-pool
+    rerun_command: /test e2e-pool
```

Comment:
Add `optional: true` before this line to make this test optional

Reply:
Thanks, done.